src/utils/formatting/stringFormatting.js

```diff
@@ -18,15 +18,15 @@ const formatValueForDisplay = (value, valueSuffix, valuePrecision) => {
 };
 
 export const formatValueFromStateAndProps = (state, ownProps) => {
-  const stockData = getSelectedStockValueForKey(state, 'stockData');
+  const stockOverviewData = getSelectedStockValueForKey(state, 'stockOverviewData');
```

Comment:
this line exceeds length 80

Reply:
you're right, but I will fix it in a future PR because there's a ton of lines throughout the project that are like this